dietpi/dietpi-software

```diff
@@ -3344,6 +3344,17 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Installing
+
+			# Link disk cache to RAM: https://github.com/Fourdee/DietPi/issues/2396
+			# - Remove previous disk cache dir or symlink
+			[[ -e /var/cache/samba ]] && rm -R /var/cache/samba
+			# - Pre-create RAM cache dir
+			mkdir -p /var/run/samba-cache
+			# - Link disk cache to RAM
+			ln -s /var/run/samba-cache /var/cache/samba
+			# - Create RAM cache dir automatically on boot
+			echo 'd /var/run/samba-cache - - - - -' > /etc/tmpfiles.d/dietpi-samba_cache.conf
```

Comment:
@MichaIng 

Nice 👍 

Reading this, does it actually make a tmpfs file, as ```/var``` is mounted to disk?
https://developers.redhat.com/blog/2016/09/20/managing-temporary-files-with-systemd-tmpfiles-on-rhel7/

Reply:
/var/run is linked to /run which is a tmpfs already.

Reply:
@Fourdee 
Jep, `tmpfiles` has nothing to do with `tmpfs` (RAMdisk). But as stated, /var/run -> /run is tmpfs as well.